ControlRoomApplication/ControlRoomApplication/GUI/RTControlFormData.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+
+namespace ControlRoomApplication.GUI.Data
+{
+	public class RTControlFormData
+	{
+		public int controlScriptIndex { get; set; }
+		public int spectraCyberScanIndex { get; set; }
+		public string frequency { get; set; }
+		public int DCGainIndex { get; set; }
+		public string IFGain { get; set; }
+		public string offsetVoltage { get; set; }
+		public int integrationStepIndex { get; set; }
+		public double speed { get; set; }
+		public bool controlledStopBool { get; set; }
+		public bool immediateStopBool { get; set; }
+		public bool manualControlEnabled { get; set; }
+		public bool freeControlEnabled { get; set; }
+		public bool scanEnabled { get; set; }
+
+
+		public RTControlFormData(int controlScriptIndex, int spectraCyberScanIndex, string frequency, int DCGainIndex, string IFGain, string offsetVoltage, 
+			int integrationStepIndex, double speed, bool controlledStopBool, bool immediateStopBool, bool manualControlEnabled, bool freeControlEnabled, bool scanEnabled)
+        {
+			this.controlScriptIndex = controlScriptIndex;
+			this.spectraCyberScanIndex = spectraCyberScanIndex;
+			this.frequency = frequency;
+			this.DCGainIndex = DCGainIndex;
+			this.IFGain = IFGain;
+			this.offsetVoltage = offsetVoltage;
+			this.integrationStepIndex = integrationStepIndex;
+			this.speed = speed;
+			this.immediateStopBool = immediateStopBool;
+			this.controlledStopBool = controlledStopBool;
+			this.manualControlEnabled = manualControlEnabled;
+			this.freeControlEnabled = freeControlEnabled;
+			this.scanEnabled = scanEnabled;
+		}
+
+		public RTControlFormData()
```

Comment:
This is missing three fields, so it'll be good to add those to initialize to 0 equivalents!